src/kirby/components/modal/action-sheet/action-sheet.component.spec.ts

```diff
@@ -17,6 +17,7 @@ import { InfiniteScrollDirective } from '../../list/directives/infinite-scroll.d
 import { CardHeaderComponent } from '../../card/card-header/card-header.component';
 import { ListFlexItemComponent } from '../../list/list-flex-item/list-flex-item.component';
 import { ButtonComponent } from '../../button/button.component';
+import { ThemeColorDirective } from '@kirbydesign/designsystem/directives/theme-color/theme-color.directive';
```

Comment:
Why is this import needed?

And why are there changes in this file?

Reply:
Needed to not break the integration test here